unity-renderer/Assets/Scripts/MainScripts/DCL/Components/CameraModeArea/CameraModeArea.cs

```diff
@@ -0,0 +1,176 @@
+using System;
+using System.Collections;
+using System.Linq;
+using DCL.Configuration;
+using DCL.Controllers;
+using DCL.Helpers;
+using DCL.Models;
+using UnityEngine;
+
+namespace DCL.Components
+{
+    public class CameraModeArea : IEntityComponent, ICameraModeArea
+    {
+        [Serializable]
+        public class Model : BaseModel
+        {
+            [Serializable]
+            public class Area
```

Comment:
We already have the `BoxTriggerArea` that represents the same for the `AvatarTriggerArea`. If we can find a nice naming and a proper location for the assemblies, we could use the same class.

Reply:
I thought about that.
The main reasons I didn't create a common type were:
1- `AvatarTriggerArea` `BoxTriggerArea` implement code, it's not just a type. So I could create a type and make `BoxTriggerArea` inherit from that, but it doesn't make de code more readable cause it will hide the `area` field
2- If we want to add more fields in one of the "Areas" and not in the other it would require a little more of work and we will find our-self in the same situation as in point 1, we will need to inherit one base type from another
So I decided to keep them separated